### ISSUE TITLE
Wrong candle data

### ISSUE THREAD

**Issue by domg73** [mikekonan/freqtrade-proxy#4]
When using freqtrade-proxy I get wrong candle data (5m timeframe) from ELON/USDT pair according to frequi.
All other pairs looks fine.
![elonusdt](https://user-images.githubusercontent.com/21331303/141820286-2bb64a28-1f00-43e0-ab91-910e144d39ce.png)

**Comment by bibinvargheset** [comment]
I had seen that

**Comment by mikekonan** [comment]
Can you check the proxy logs? Is there anything with that pair?

**Comment by domg73** [comment]
The log is clean... nothing about that pair.

**Comment by mikekonan** [comment]
So freqtrade did not make requests to get candle data for this pair

**Comment by domg73** [comment]
Without proxy candle data for that pair is ok, so freqtrade actually make requests for that pair.

**Comment by Smidelis** [comment]
I've got the same issue with NFT/USDT
![image](https://user-images.githubusercontent.com/74504278/142182282-3ddbf936-181c-4d33-a042-516b11a3d49a.png)
Proxy is already updated to v1.0.7

**Comment by Smidelis** [comment]
```
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=STRONG-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=PBX-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=BLOK-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=HERO-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=ICP-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=SENSO-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=ETH-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=BNB-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=KDA-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=OMG-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=LTO-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=ENS-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=RNDR-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=JASMY-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=XMR-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=VRA-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=FTM-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=HTR-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=CHR-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=WIN-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=NEAR-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=MANA-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=NFT-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=QRDO-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=XTM-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=BTC-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=MTV-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:03Z" level=info msg="processing request - /kucoin/api/v1/market/candles?symbol=CPOOL-USDT&type=1min&startAt=1637116623&endAt=1637146623" caller="/src/proxy/kucoin/kucoin.go:226 (*kucoin).Start.func1"
time="2021-11-17T10:57:04Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=HAI-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:06Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:07Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=HAI-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:08Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:08Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=CPOOL-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:08Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=GALAX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:08Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=IOTX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:09Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:09Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=CPOOL-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:10Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=GALAX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:10Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=IOTX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:10Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=HAI-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:12Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:12Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:12Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=CPOOL-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:13Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=GALAX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:13Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=IOTX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:13Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=HAI-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:15Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:15Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=CPOOL-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:16Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=GALAX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:16Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=IOTX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:16Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=HAI-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:18Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:18Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=NFT-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:18Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=CPOOL-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:19Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=GALAX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:19Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=HAI-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
time="2021-11-17T10:57:20Z" level=info msg="proxying over - /kucoin/api/v1/market/stats?symbol=IOTX-USDT" caller="/src/proxy/kucoin/kucoin.go:265 (*kucoin).Start.func2"
```
Logs look fine as far as i can judge.

**Comment by mikekonan** [comment]
Will investigate it later

**Comment by mikekonan** [comment]
Fixed in v1.0.8
